src/stories/6.network.stories.jsx

```diff
@@ -174,6 +174,21 @@ const testRemote = () => {
 const testLocal = () => {
   return { ...mainLocal(), network: 'ropsten' }
 }
+const testLocalSyncing = () => {
+  return {
+    ...testLocal(),
```

Comment:
Can `testRemote` and `testLocal` (and other similar utilities) be objects instead of functions? Usage simplifies a bit and unsure why they would need to be functions on quick look.

Reply:
They were objects, but I turned them into functions because when testing in storybook I wanted the timestamp of the block to be from the moment of the click on the story, not from page load which is what was happening.

So for example this is what I did:

```js
const mainRemote = () => {
  return {
    active: 'remote',
    network: 'main',
    changingNetwork: false,
    remote: {
      client: 'infura',
      blockNumber: 100000,
      timestamp: moment()
        .subtract(10, 'seconds')
        .unix()
    },
    local: { ...initial.local }
  }
}```